### PR TITLE
[Admin][Payments] Payments index with filters and sorting

### PR DESCRIPTION
**Pull request by mpysiak**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | bootstrap-admin-panel                  |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | n/a                     |
| License         | MIT                                                          |

### CODE REVIEW

File: features/admin/payment/managing_payments/browsing_payments.feature [hunk 1]
```diff
@@ -12,21 +12,37 @@ Feature: Browsing payments
...
-    @todo @ui @api
+    @ui @api
     Scenario: Payments are sorted by newest as default
         Given there is an "#00000002" order with "Apple" product ordered later
         When I browse payments
         Then I should see payment for the "#00000002" order as 1st in the list
         And I should see payment for the "#00000001" order as 2nd in the list
+
+    @ui @api-todo
+    Scenario: Payments are sorted by date in ascending order
```
Comment (Wojdylak) [inline features/admin/payment/managing_payments/browsing_payments.feature]:
```suggestion
    Scenario: Sorting payments by date in ascending order
```

**Review by Wojdylak — APPROVED** [review]

File: features/admin/payment/managing_payments/browsing_payments.feature (hunk 1 above)
Comment (mpysiak) [reply features/admin/payment/managing_payments/browsing_payments.feature]: Changed

File: features/admin/payment/managing_payments/browsing_payments.feature [hunk 2]
```diff
@@ -12,21 +12,37 @@ Feature: Browsing payments
...
+        Then I should see payment for the "#00000001" order as 1st in the list
+        And I should see payment for the "#00000002" order as 2nd in the list
+
+    @ui @api-todo
+    Scenario: Sorting payments by date in descending order again
+        Given there is an "#00000002" order with "Apple" product ordered later
+        When I browse payments
+        And I sort payments by date in descending order
+        Then I should see payment for the "#00000002" order as 1st in the list
+        And I should see payment for the "#00000001" order as 2nd in the list
```
Comment (GSadee) [inline features/admin/payment/managing_payments/browsing_payments.feature]: These new scenarios for sorting can be extracted into a separate file

Comment (mpysiak) [reply features/admin/payment/managing_payments/browsing_payments.feature]: I've extracted these tests

File: features/admin/payment/managing_payments/sorting_payments.feature [hunk 3]
```diff
@@ -0,0 +1,28 @@
+@managing_payments
+Feature: Sorting payments
+    In order to manage all payments regardlessly of orders
```
Comment (Rafikooo) [inline features/admin/payment/managing_payments/sorting_payments.feature]:
```suggestion
    In order to manage all payments regardless of orders
```

**Review by Rafikooo — APPROVED** [review]

**Review by Rafikooo — APPROVED** [review]

**Comment by Wojdylak** [comment]
Thank you, @mpysiak :tada:

**Pull request merged**